platformio.ini

```diff
@@ -52,7 +52,7 @@ src_build_flags =
 	-Wno-unused-parameter
 ;	-Wold-style-cast
 	-Wlogical-op
-	-Wuseless-cast
+;	-Wuseless-cast
```

Comment:
Why remove this warning?

Reply:
There was some case I couldn't get rid of, but looks like it's gone now. Reverted.